conf/etc/v2/event_analyzer.service

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+[Unit]
+Description=HA event analyzer daemon process
+After=syslog.target
+
+[Service]
+Type=simple
+ExecStartPre=/usr/bin/sleep 5
+ExecStart=/usr/bin/event_analyzerd
+TimeoutStopSec=7sec
+# TODO: user to be changed to hauser
```

Comment:
Why this TODO?
Is there a confusion that the user should be hauser or root?

Reply:
The user should be hauser. 
However hauser is currently not created in HA repository, so I have added root here 
Once hauser gets created we can change this line